pkg/controller/pxc/upgrade.go

```diff
@@ -283,15 +283,19 @@ func (r *ReconcilePerconaXtraDBCluster) proxyDB(cr *api.PerconaXtraDBCluster) (q
 	var database queries.Database
 	var user, host string
 	var port, proxySize int32
+
+	port = 3306
+	if cr.CompareVersionWith("1.6.0") >= 0 {
+		port = 33062
```

Comment:
please revert 6032 port for ProxySQL.
it is needed here for proper connection to ProxySQL itself

Reply:
Thanks